### ISSUE TITLE
Bug: neovim version and other info as soon as nvim is opened is not being displayed

### ISSUE THREAD

**Issue by SyedFasiuddin** [nvim-lualine/lualine.nvim#983]
### Self Checks

- [x] I'm using the latest lualine.
- [x] I didn't find the issue in existing issues or PRs.

### How to reproduce the problem
```lua
require("lualine").setup({})
```

### Expected behaviour
Had to remove lualine to show that without lualine the nvim version and other info is being displayed.
<img width="780" alt="image" src="https://user-images.githubusercontent.com/66054777/222390762-d3744f2b-73e2-4338-8901-8d55f6b688ef.png">

### Actual behaviour
<img width="787" alt="image" src="https://user-images.githubusercontent.com/66054777/222390901-f2ecfd36-3585-4f5a-b734-04d08e6962e8.png">

### Minimal config to reproduce the issue
```lua
require("lualine").setup({})
```

**Comment by supermacro** [comment]
FWIW, I get a "flash" of the startup page: https://www.loom.com/share/cda772dd272240dbad05938096a272f4

**Comment by SyedFasiuddin** [comment]
In neovim version 0.9.0 this seems to be fixed (there was no change in my config)

**Comment by NStefan002** [comment]
I don't know if this helps, but I solved the 'flash of startup page' with the following code:
```lua
-- If the current buffer is a 'No Name' buffer, display the startup page
if vim.api.nvim_buf_get_name(vim.api.nvim_get_current_buf()) == '' then
    vim.cmd('intro')
end
```

Note: I am using Neovim version 0.9.0 and lualine is up to date.

**Comment by AndreM222** [comment]
I also had the same problem. this has not been patched right?

**Comment by SyedFasiuddin** [comment]
right

**Comment by leandrocunha** [comment]
I have the same, still waiting for some resolution

**Comment by ryan-talus** [comment]
I also have this issue, but there is a workaround; if you change section separators to an empty string, you get the message back. 

```lua
require('lualine').setup {
    options = {
      -- disabling section separators fixes
      -- the disappearing start screen issue
      section_separators = '',
    }
  }
```

<img width="714" alt="image" src="https://github.com/nvim-lualine/lualine.nvim/assets/61983809/98b8f0c0-de12-43e3-adf9-4101f60bea53">

Personally I don't use this because I like the angled separators, because they match my tmux theme separators, but this is a very curious bug, I really wonder what it is about setting separators which (seems to) cause it.

**Comment by SyedFasiuddin** [comment]
> if you change section separators to an empty string, you get the message back.

I have `section_separators = ""` in my config but still I do not get the startup screen.

**Comment by ryan-talus** [comment]
> if you change section separators to an empty string, you get the message back.
> I have `section_separators = ""` in my config but still I do not get the startup screen.

Seems even more mysterious! 

You may need to disable other plugins too, when I started troubleshooting this I had several which hid the startup screen. I actually just now set some of the others to lazy load and switched my config to feline.nvim which let me style it how I wanted while still showing the startup screen.

**Comment by SyedFasiuddin** [comment]
> You may need to disable other plugins too
> 
yes, in the original post where I mentioned "minimal config" lualine was the only plugin that was active.

**Comment by leandrocunha** [comment]
> I also have this issue, but there is a workaround; if you change section separators to an empty string, you get the message back.
> 
> ```lua
> require('lualine').setup {
>     options = {
>       -- disabling section separators fixes
>       -- the disappearing start screen issue
>       section_separators = '',
>     }
>   }
> ```
> 
> <img alt="image" width="714" src="https://user-images.githubusercontent.com/61983809/282269153-98b8f0c0-de12-43e3-adf9-4101f60bea53.png">
> Personally I don't use this because I like the angled separators, because they match my tmux theme separators, but this is a very curious bug, I really wonder what it is about setting separators which (seems to) cause it.

works for me!

**Comment by helins** [comment]
Odd, I too can solve it by setting `section_separators` (using Neovim 0.9.5).
However, `flash.nvim` also cause the same issue, there is something else going on...

**Comment by tgallacher** [comment]
Also experiencing this issue, nvim 0.9.5. As a result of something in this plugin, and a few others. 

Issue seems to be in plugins which cause any "redraw" of the nvim UI. This results in the startup info disappearing (`:h :intro`).

In some cases you can workaround this buy lazy loading the plugin into the `VimEnter` event, but sadly that doesn't work for me. And, I also have custom separators set, so the above workaround is a no go for me either 😞
